daft/daft/image.pyi

```diff
@@ -0,0 +1,6 @@
+from daft.daft import ImageFormat, ImageMode, PySeries
+
+def decode(s: PySeries, raise_error_on_failure: bool, mode: ImageMode | None = None) -> PySeries: ...
+def encode(s: PySeries, image_format: ImageFormat) -> PySeries: ...
+def resize(s: PySeries, w: int, h: int) -> PySeries: ...
+def to_mode(s: PySeries, mode: ImageMode) -> PySeries: ...
```

Comment:
Nice, I like the idea of submodules here so we don't have to stuff everything in `daft.pyi`